rust/hyperlane-base/src/settings/staking_config.rs

```diff
@@ -0,0 +1,18 @@
+use hyperlane_core::H160;
+use std::collections::HashMap;
+
+#[derive(Clone, Debug, Default)]
+pub struct StakingConf {
+    /// Address of the HyperlaneServiceManager contract
+    pub service_managers: HashMap<u32, H160>,
```

Comment:
will there be stuff in here other than the service_managers?

Reply:
Yes, I imagine we'll have more config once we have native staking which will also have some form of registration announcement in a json form.